configs/agent/sample.toml

```diff
@@ -66,8 +66,10 @@ scaling-group = "default"
 # This directory is used to store the last registry file.
 # var-base-path = "/var/lib/backend.ai"
 
-# Optional, directory to look up abusing kernel reports generated by watchers
-abuse-report-path = "/tmp/backend.ai/abuse-reports"
+# The directory to look up abusing kernel reports generated by watchers
```

Comment:
Can you write down more about those flags? Explaining about the matrix on PR description will be enough.

Reply:
Updated the docs.